.circleci/config.yml

```diff
@@ -2965,6 +3127,36 @@ workflows:
             tags:
               only:
                 - /v[0-9]+(\.[0-9]+)*-rc[0-9]+/
+      - binary_windows_build:
```

Comment:
these are generated because it was not blocklisted in windows build 

please add them in:
https://github.com/pytorch/pytorch/blob/master/.circleci/cimodel/data/binary_build_data.py#L57

Reply:
In fact there are several exclude "rocm3.7" checks in 
https://github.com/pytorch/pytorch/blob/master/.circleci/cimodel/data/binary_build_data.py 

Please change those `rocm3.7` checks to `rocm*` checkers

Reply:
Fixed.